thunder/torch/__init__.py

```diff
@@ -1801,6 +1801,17 @@ def gelu(a: TensorProxy, /, *, approximate: str = "none") -> TensorLike:
         raise ValueError(f"gelu does not support the approximate={approximate} argument")
 
 
+@torchsymbol(torch.nn.functional.leaky_relu, is_method=False)
+def leaky_relu(a: TensorProxy, /, negative_slope=0.01, inplace: bool = False) -> TensorLike:
```

Comment:
Nit: `negative_slope: Number = 0.01` or `negative_slope: float` if `negative_slope` has to be a float

Reply:
Thanks, should be a float.